package.json

```
@@ -35,6 +36,7 @@
     "test:coverage": "jest --coverage",
     "tsc": "tsc",
     "validate": "yarn tsc && yarn lint && yarn test",
+    "validate-dist": "yarn build:all && yarn workspace @guardian/source-validate-dist validate",
```

```suggestion
    "validate-dist": "yarn build:all && yarn workspace validate-dist validate",
```

(ignore this if you don't want to go with the suggestion on the package's own `package.json` below)

---

👍 This should also be done in https://github.com/guardian/source/pull/985/commits/b307fe3df664048fe31f4e707a5cf5089a08bad6